wp-content/themes/access/views/single-page.twig

```diff
@@ -12,10 +12,13 @@
   </div>
   {% endif %}
 
-  <article>
+  <article itemtype="https://schema.org/mainEntityOfPage">
```

Comment:
`itemtype`s should be `itemprop` within the scope of the `WebPage` schema and you can drop the url.

```
itemprop="mainEntityOfPage">
```